packages/web-api-scan-runner/src/crawl-runner/crawl-runner.ts

```diff
@@ -19,14 +18,12 @@ export class CrawlRunner {
         @inject(crawlerIocTypes.CrawlerProvider) private readonly getCrawler: CrawlerProvider,
```

Comment:
Confusing name alias. If `iocTypes` is exported from cli it should be named as `crawlerIocTypes` in cli instead of aliasing in `import`